kostyor/db/api.py

```diff
@@ -10,24 +12,34 @@
 from sqlalchemy.orm import scoped_session, sessionmaker
 
 
-# TODO (sc68cal) save the database file in a configurable location
-engine = create_engine('sqlite:////tmp/test.db', convert_unicode=True)
-db_session = scoped_session(sessionmaker(autocommit=False,
-                                         autoflush=False,
-                                         bind=engine))
+@contextmanager
+def create_db_session():
```

Comment:
Hm.. What the reason of creating a `scoped_session` inside a context manager? It makes little sense to me since `scoped_session` is tied to thread and there will be no problem with concurrent requests from thread. It even works fine with `gevent`/`eventlet` approach (since they patches `ThreadLocal`). If I'm not mistaken, it's a common practice to define `scoped_session` globally [[1]]. Alternatively, we can return a usual `Session` object from the context manager instead. However, in my opinion having one session per-thread makes things simple and easy to use. And it'll allow to implement "one db transaction per one http request" feature which is a good thing.

[1]: http://docs.sqlalchemy.org/en/latest/orm/contextual.html#thread-local-scope